agent/bench-scripts/pbench-uperf

```diff
@@ -147,8 +149,10 @@ function gen_xml {
 		echo "    <transaction iterations=\"1\">"
 		echo "      <flowop type=\"disconnect\" />"
 		echo "    </transaction>"
-		;;
-		maerts|bidirec)
+		echo "  </group>"
+		;;&
```

Comment:
According to @atheurer the behavior or `;;&` is not consistent/portable across different versions of Bash.

Reply:
Would you agree that given we are moving to Python 3 in the future for all this, and that our version of bash for our target environments is constrained to Fedora derived distributions, that this seems like the simplest and most straightforward change for the v0.68 release?

Reply:
All I know is that I've been told this isn't portable.  I have no idea what the set of Bash versions that this does and does not work on is, but choosing to do something that you know will break somewhere seems far from "simplest and most straight forward".

Reply:
Good points.

From what I have read [1], the `;;&` syntax is not POSIX, is available in `zsh` and `ksh` (derived from there), but according to the `bash` docs [2] it is well documented in `bash`.

Seems like this is a safe syntax to use with bash. 

[1] https://unix.stackexchange.com/questions/75354/can-bash-case-statements-cascade
[2] https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Conditional-Constructs